README.md

```diff
@@ -1,104 +1,159 @@
+
+We now bundle the Cohort, ConditionSet, and any other extra parameters into a [_Strategy_](https://docs.threshold.network/app-development/threshold-access-control-tac/references/strategy)_:_
+
+```javascript
+import { Strategy } from '@nucypher/nucypher-ts';
+
+const newStrategy = Strategy.create(
+  newCohort,
+  conditions
+);
 ```
 
-## Publishing
+Next, we deploy this Strategy to the Threshold Network:
 
-Publish a new release on NPM.
+```typescript
+import detectEthereumProvider from '@metamask/detect-provider';
+import providers from 'ethers';
 
-Pay attention to output of these commands and fix your release if needed.
+const MMprovider = await detectEthereumProvider();
+const goerli = providers.providers.getNetwork('Goerli');
 
-To build and publish a release, run
+if (MMprovider) {
+  const web3Provider = new providers.providers.Web3Provider(
+    MMprovider,
+    goerli
+  );
+  const newDeployed = await newStrategy.deploy('test', web3Provider);
+} 
+```
 
-```bash
-yarn prepare-release
-# Or, to publish an alpha release
-yarn prepare-release:alpha
+> :warning: Deploying a Strategy requires writing to the blockchain. This requires a funded wallet and connection to the blockchain via a `provider`(e.g. MetaMask).
+
+## 6. Encrypt the plaintext
+
+We're now able to encrypt data to this newly deployed Strategy – which implies future access to this data will be based on ownership of the previously specified NFT, and nothing else. We'll now encrypt a plaintext using the encryptor object:
+
+```javascript
+const encrypter = newDeployed.encrypter;
+
+const plaintext = 'this is a secret';
+const encryptedMessageKit = encrypter.encryptMessage(plaintext);
 ```
 
-Follow instructions from the command output to finalize the process.
+## 7. Request decryption rights
 
-## Examples
+Finally, we'll test the access control service by submitting a request to the network:
 
-See `./test` directory for usage examples.
+```javascript
+const decrypter = newDeployed.decrypter;
 
-See `./examples` directory for examples of browser integration.
+const decryptedMessage = await decrypter.retrieveAndDecrypt([
+  encryptedMessageKit,
+]);
+```
 
-See `./examples/api-example.ts` for an abridged API example.
+At decryption time, the requester will be asked to verify their address by signing a message in MetaMask. If they own the correct NFT, the message will decrypt successfully.
 
-See [`nucypher-ts-demo`](https://github.com/nucypher/nucypher-ts-demo) for usage example with React app.
+For more guidance on Cohort, Condition and Strategy object reuse and customization, check out the [References](https://docs.threshold.network/app-development/threshold-access-control-tac/references) page.
 
-### Using Threshold Decryption
 
-There are several schemes available for the user.
-They follow a naming convention of `<NETWORK>-<M>-of-<N>`, where `M` is the threshold required, and `N` is the total size of the cohort.
-`NETWORK` is either `mainnet` or `ibex`.
+# Contributing
```

Comment:
We could move this section to a separate [CONTRIBUTING file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors).